src/main/kotlin/net/maizegenetics/phgv2/pathing/BuildKmerIndex.kt

```diff
@@ -39,25 +39,27 @@ import kotlin.time.measureTimedValue
  *
  * To use this class, ...
  */
-class BuildKmerIndex: CliktCommand(help="Create a kmer index for a HaplotypeGraph") {
+class BuildKmerIndex: CliktCommand(help="Create a kmer index for a HaplotypeGraph. By default the file will be written " +
+        "to <hvcfDir>/kmerIndex.txt") {
 
     private val myLogger = LogManager.getLogger(BuildKmerIndex::class.java)
 
     val dbPath by option(help = "Tile DB URI")
         .required() //Needs to be required now due to the agc archive
 
     val maxHaplotypeProportion by option("-p", "--maxHapProportion", help = "only kmers mapping to less than or " +
-            "equal to maxHapProportion of haplotypes in a reference range will be retained.")
+            "equal to maxHapProportion of haplotypes in a reference range will be retained.Defualt = 0.75")
```

Comment:
Typo with Default here.

Reply:
When asking for help, Clikt might even provide the default automatically without it needing to be in the description.

Reply:
Clickt --help does not provide either default values or information about whether or not a parameter is required.